### PR TITLE
Btfs 1924 2

### PR DESCRIPTION
**Pull request by steveyeom**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,10 @@
-module github.com/ipfs/go-log/v2
+module github.com/TRON-US/go-log/v2
 
-require go.uber.org/zap v1.14.1
+require (
+	github.com/TRON-US/go-btfs-collect-client v0.0.1
+	go.uber.org/zap v1.4.1
```
Comment (taiyangc) [inline go.mod]: should be v1.14.1 ?

**Review by taiyangc — COMMENTED** [review]
Could you get rid of the IPFS related keywords in README.md

File: go.mod (hunk 1 above)
Comment (steveyeom) [reply go.mod]: Updated.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+[![](https://img.shields.io/badge/project-BTFS-blue.svg?style=flat-square)](http://btfs.io/)
+[![](https://img.shields.io/badge/freenode-%23btfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23btfs)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
-[![GoDoc](https://godoc.org/github.com/ipfs/go-log?status.svg)](https://godoc.org/github.com/ipfs/go-log)
-[![Build Status](https://travis-ci.org/ipfs/go-log.svg?branch=master)](https://travis-ci.org/ipfs/go-log)
+[![GoDoc](https://godoc.org/github.com/TRON-US/go-log?status.svg)](https://godoc.org/github.com/TRON-US/go-log)
+[![Build Status](https://travis-ci.org/TRON-US/go-log.svg?branch=master)](https://travis-ci.org/TRON-US/go-log)
 
-<!---[![Coverage Status](https://coveralls.io/repos/github/ipfs/go-log/badge.svg?branch=master)](https://coveralls.io/github/ipfs/go-log?branch=master)--->
+<!---[![Coverage Status](https://coveralls.io/repos/github/TRON-US/go-log/badge.svg?branch=master)](https://coveralls.io/github/TRON-US/go-log?branch=master)--->
```
Comment (taiyangc) [inline README.md]: can you remove all badges, thx

**Review by taiyangc — APPROVED** [review]

**Pull request merged**